src/Cubie/Moves.hs

```diff
@@ -0,0 +1,26 @@
+module Cubie.Moves
```

Comment:
@SaxJ what do you think to this pattern where there is a module `Cubie.Moves` and submodules `Cubie.Moves.Edges` and therefore a file at `Cubie/Moves.hs` and a directory `Cubie/Moves/`. I've already found it annoying for tab complete in CLI but what have you seen or used in other projects?

Reply:
Also, I'm thinking (at least initially) that we keep the permutation functions separate for edges and corners as in this PR and we can add moves that decompose the Cube type and pass the vectors on to the correct functions from the submodules. So, after the #14 is merged, each move can be added to this file. Any better ideas or objections?